src/components/Layout/MainHeader/MainHeader.tsx

```diff
@@ -40,7 +46,12 @@ export const MainHeader = () => {
       )}
     >
       <div className="h-16 flex justify-between items-center max-w-[1200px] w-full mx-auto">
-        <img src="/freshnesecom.svg" alt="" />
+        <img
+          src="/freshnesecom.svg"
+          alt=""
+          onClick={() => goToMainPage()}
```

Comment:
Please fix the previous comments and update with develop this PR

Reply:
done